$OpenBSD: patch-configure_ac,v 1.1.1.1 2014/04/13 07:10:12 rpointel Exp $

SOVERSION defaults to 1.0. SHARED_LIBS, however, could be changed
at any point.  Ensure they are on sync in case INSTSONAME is used
by a third party.

--- configure.ac.orig	Mon Mar 17 03:31:31 2014
+++ configure.ac	Sat Apr  5 09:24:16 2014
@@ -961,6 +961,9 @@ if test $enable_shared = "yes"; then
 	      FreeBSD*)
 		SOVERSION=`echo $SOVERSION|cut -d "." -f 1`
 		;;
+	      OpenBSD*)
+		SOVERSION=${LIBpython3.4m_VERSION}
+		;;
 	  esac
 	  INSTSONAME="$LDLIBRARY".$SOVERSION
 	  if test "$with_pydebug" != yes
